aerleon/lib/naming.py

```diff
@@ -663,111 +706,65 @@ def ParseNetworkList(self, data: List[str]) -> None:
           data: array of text lines containing net definitions.
 
         """
-        for line in data:
-            self._ParseLine(line, DEF_TYPE_NETWORKS)
-
-    def _ParseLine(self, line: str, definition_type: str) -> None:
```

Comment:
I'm concerned it might not be safe to remove `Naming._ParseLine` in case network automation tools or scripts are using it to load data one line at a time. I suspect it might be in use because it is easy to loop over your input and call `Naming._ParseLine` each time than to construct a whole file and call `Naming.ParseNetworkList` (even though it has an underscore name).